docs/cimpl6LanguageReference.md

```diff
@@ -308,29 +290,27 @@ The version details the _major_ and _minor_ version of the grammar the file conf
 
 ### Group
 
-The `Group` keyword is used to declare a referenceable collection of one or more properties, similar to a **complex data type** in FHIR. For details, see [Class File](#class-file).
+The `Group` keyword is used to declare a reusable collection of one or more properties, similar to the [complex data types](https://www.hl7.org/fhir/datatypes.html#complex) in FHIR. For details on using the `Group` keyword, see [Class File](#class-file).
 
 Example:
 
 `Group: Dosage`
 
 ### Namespace
 
-The `Namespace` keyword defines the association of the file with a namespace, and is required in `Class`, `ValueSet` and `Map` files. The [namespace value](#namespace-names) can be any number of lowercase period-delimited words.
+The `Namespace` keyword defines the association of the file with a namespace, and is required in `Class`, `ValueSet` and `Map` files. The namespace name should follow [naming conventions](#namespace-names) discussed earlier.
 
 Examples:
 
 `Namespace: odh`
 
 `Namespace: onco.core`
 
->**Note:** While it is best practice to use only lower case letters, the strict requirements only require the word to begin with a lowercase letter. The rest of the namespace allows for mixed casing and hyphens.
-
 ### Parent
 
-Through the `Parent` keyword, CIMPL provides a mechanism for basing a class on another class, the child inheriting all properties and constraints from the parent. The child class can then extend information inherited from the parent by adding additional properties and applying additional constraints. Additionally, maps are inherited alongside properties. At most one parent class can be specified. The `Parent` declaration is optional.
+Through the `Parent` keyword, CIMPL provides a mechanism for deriving a class from another class. The child class inherits all properties and constraints from the parent. The child class can then add additional properties and constraints. Additionally, maps are inherited alongside properties. At most one parent class can be specified. The `Parent` declaration is optional.
 
-There are restrictions on which building blocks can inherit from other building blocks. The rules can be summarized as _like inherits from like_:
+There are restrictions on which classes can inherit from other classes. The rules can be summarized as _like inherits from like_ (noting that `Abstract` is an `Entry` that is non-instantiable):
```

Comment:
Suggest providing more info about what we're trying to get across with the note about Abstract.

Reply:
Clarified. "Note that `Abstract` is _like_ an `Entry`, except that `Abstract` cannot be instantiated."